gfx/video_crt_switch.c

```diff
@@ -235,6 +265,7 @@ void crt_destroy_modes(videocrt_switch_t *p_switch)
    if (p_switch->sr2_active == true)
    {
       p_switch->sr2_active = false;
+      free(p_switch->core_name);
```

Comment:
For general safety, could you please set `p_switch->core_name` to `NULL` after freeing the pointer?

Reply:
Please see my above comment regarding changes to the variable `core_name`, which is no longer part of the original struct and is instead a static variable. This `free` command is no longer needed.

Changes are present on the just-amended commit.